pym/bob/invoker.py

```diff
@@ -177,6 +178,10 @@ async def __runCommand(self, args, cwd, stdout=None, stderr=None,
             _env.update(env)
             env = _env
 
+        if len(self.__makeFds) == 2:
+            env["MAKEFLAGS"] = "".join([env.get("MAKEFLAGS") if env.get("MAKEFLAGS") is not None else ""]) + \
+                "-j --jobserver-auth=" + ",".join([str(fd) for fd in self.__makeFds])
```

Comment:
Shouldn't there be a space between multiple options? I also thought that make forwards the number of jobs, e.g. `-j4` instead of an unbounded `-j`.